.github/workflows/gradle-build.yml

```diff
@@ -14,11 +14,21 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          java-version: 11
```

Comment:
out of curiosity - why not 14? android reasons or other?

Reply:
java 11 is the only LTS until 17